src/server.js

```diff
@@ -36,6 +36,11 @@ module.exports = function() {
 		.set("view engine", "html")
 		.set("views", path.join(__dirname, "..", "client"));
 
+	// Express will handle X-Forwarded-For header for us
+	if (Helper.config.reverseProxy) {
+		app.enable("trust proxy");
```

Comment:
As far as I can see from the [docs](https://expressjs.com/en/guide/behind-proxies.html), doing this will take the default IP and not the forward one. I think you'll want to do app.enable("trust proxy", true) which will take the first from x-forwarded-for.